dev/run-tests.py

```diff
@@ -500,12 +500,12 @@ def main():
         else:
             print("Cannot install SparkR as R was not found in PATH")
 
-    if os.environ.get("AMPLAB_JENKINS"):
+    if os.environ.get("JENKINS"):
```

Comment:
No biggie but I wonder if it's better to have a different prefix instead.. because at least JENKINS might be common enough and already used for other purposes in other systems.

Reply:
Oh, right. Let me use `SPARK_` then.